packages/shared/providers.yaml

```diff
@@ -5578,6 +5578,27 @@ twinfield:
     disable_pkce: true
     docs: https://docs.nango.dev/integrations/all/twinfield
 
+twenty-crm:
+    display_name: Twenty CRM
+    categories:
+        - crm
+    auth_mode: API_KEY
+    proxy:
+        base_url: https://api.twenty.com/rest
+        headers:
+            Authorization: Bearer ${apiKey}
+        retry:
+            after: 'Retry-After'
+        verification:
+            method: GET
+            endpoint: /companies
+    docs: https://docs.nango.dev/integrations/all/twenty-crm
+    credentials:
+        apiKey:
+            type: string
+            title: API Key
+            description: The API key for your Twenty CRM account
```

Comment:
If you can add pattern for verification 👍🏻

Reply:
Added with https://github.com/NangoHQ/nango/pull/2946/commits/151def42a6f0c0736e153898436c09f0f203ab2f